pkg/controller/quayecosystem/externalaccess/loadbalancer.go

```diff
@@ -0,0 +1,144 @@
+package externalaccess
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/redhat-cop/quay-operator/pkg/controller/quayecosystem/logging"
+	"github.com/redhat-cop/quay-operator/pkg/controller/quayecosystem/resources"
+	"github.com/redhat-cop/quay-operator/pkg/controller/quayecosystem/utils"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	tools_watch "k8s.io/client-go/tools/watch"
+)
+
+type LoadBalancerExternalAccess struct {
+	QuayConfiguration *resources.QuayConfiguration
+	K8sClient         kubernetes.Interface
+}
+
+func (r *LoadBalancerExternalAccess) ManageQuayExternalAccess(meta metav1.ObjectMeta) error {
+	serviceName := resources.GetQuayResourcesName(r.QuayConfiguration.QuayEcosystem)
+
+	hostname, err := r.getHostnameFromExternalName(serviceName)
+
+	if err != nil {
+		return err
+	}
+
+	r.QuayConfiguration.QuayHostname = hostname
+
+	return nil
+
+}
+
+func (r *LoadBalancerExternalAccess) ManageQuayConfigExternalAccess(meta metav1.ObjectMeta) error {
```

Comment:
I noticed that `meta` is not used in this method, among others. Is that just done to keep the interface generic/consistent?

Reply:
yes it is a required method signature